lib/src/utils.dart

```diff
@@ -17,6 +17,9 @@ import 'package:sass/src/ast/node.dart';
 import 'io.dart';
 import 'patch.dart';
 
+T assertNotNull<T>(T? object) =>
+    object ?? (throw AssertionError('Unexpected null value.'));
```

Comment:
Done.

All but one use of this was for a variable that's initialized in `visitStylesheet` of a `MigrationVisitor`, so I added a method there. `_configuredVariables` is initialized when a dependency is visited, but since it's the only case like that I just threw the error inline.